content/en/docs/contribute/new-content/new-features.md

```diff
@@ -98,7 +98,8 @@ deadlines.
 1. Open a pull request against the
 `dev-{{< skew nextMinorVersion >}}` branch in the `kubernetes/website` repository, with a small
 commit that you will amend later.
-2. Use the Prow command `/milestone {{< skew nextMinorVersion >}}` to
+2. Edit the pull request description to include links to `k/k` PR(s) and `k/enhancement` issue(s).
+3. Use the Prow command `/milestone {{< skew nextMinorVersion >}}` to
```

Comment:
What privilege is required to add the milestone label?

Reply:
AFAIK you need to be part of the `website-milestone-maintainer` group to be able to be able to add the milestone. 

But, I believe the doc is saying people should use the command to notify the docs person managing the release not necessary to add the milestone.

Reply:
Wondering if we can include this as a comment in the PR to notify whoever is able to add milestones?

Reply:
Hi Irvi, are you saying that you would like to see this information in the PR template? 

I currently have the link to this doc in the PR template, and I agree with Karen that this may be a lot cleaner approach then to list the requirements in the PR template. What do you think?

Reply:
Oh sorry I didn’t saw the link, it’s okay then! Thanks for the clarification.

Reply:
It would be cool if we had a `/milestone next` command